### PR TITLE
Bug 1667791 - Update NimbusExperiment schema

### PR DESCRIPTION
**Pull request by piatra**
Schema got a slight update, the `arguments` field is now gone so the `validate.py` script also needs an update.
- [x] validate.py update

### CODE REVIEW

**Comment by piatra** [comment]
I've reverted the PR that used 2 experiment schemas in parallel and now only validate against the new one. I added a dummy message for sanity check.

File: schema/messaging-experiments.schema.json [hunk 1]
```diff
@@ -1,93 +1,171 @@
...
+            "randomizationUnit",
+            "namespace",
+            "start",
+            "count",
+            "total"
+          ],
+          "additionalProperties": false,
+          "description": "Bucketing configuration"
+        },
+        "probeSets": {
```
Comment (ncloudioj) [inline schema/messaging-experiments.schema.json:67]: [Non-issue] Interesting, I wonder how this works.

File: scripts/validate.py [hunk 2]
```diff
@@ -30,7 +30,6 @@
     "cfr-fxa": "schema/cfr-fxa.schema.json",
     "cfr-heartbeat": "schema/cfr-heartbeat.schema.json",
     "messaging-experiments": "schema/messaging-experiments.schema.json",
-    "messaging-experiments-82": "schema/messaging-experiments-82.schema.json",
```
Comment (ncloudioj) [inline scripts/validate.py:33]: Shall we drop this schema file as well?

File: scripts/validate.py [hunk 3]
```diff
@@ -226,28 +225,17 @@ def validate_experiment(item):
...
-                    try:
-                        print("Validate {} with Experiments schema 82+".format(item["id"]))
-                        jsonschema.validate(instance=item, schema=new_experiments_schema)
-                    except ValidationError as err:
-                        match = best_match([err])
-                        print("Validation error against the new experiments schema: {}"
-                              .format(match.message))
-                        jsonschema.validate(instance=item, schema=schema)
-                    # If it's an experiment we want to evaluate the branches
+                jsonschema.validate(instance=item, schema=schema)
```
Comment (ncloudioj) [inline scripts/validate.py:235]: Nice cleanup 👍

**Review by ncloudioj — APPROVED** [review]
Looks great, just one comment for a potential file drop.

Thanks!

**Pull request merged**